man/openmp-utils.Rd

```diff
@@ -41,17 +41,18 @@
     \item\file{cj.c} - \code{\link{CJ}()}
     \item\file{coalesce.c} - \code{\link{fcoalesce}()}
     \item\file{fifelse.c} - \code{\link{fifelse}()}
-    \item\file{fread.c} - \code{\link{fread}()}
+    \item\file{fread.c} - \code{\link{fread}(). Parallelized across row-based chunks of the file.}
     \item\file{forder.c}, \file{fsort.c}, and \file{reorder.c} - \code{\link{forder}()} and related
     \item\file{froll.c}, \file{frolladaptive.c}, and \file{frollR.c} - \code{\link{froll}()} and family
-    \item\file{fwrite.c} - \code{\link{fwrite}()}
-    \item\file{gsumm.c} - GForce in various places, see \link{GForce}
+    \item\file{fwrite.c} - \code{\link{fwrite}(). Parallelized across rows.}
+    \item\file{gsumm.c} - GForce in various places, see \link{GForce}. Parallelized across groups.
```

Comment:
for me this is sufficient documentation of whether the parallelization happens on rows or columns

Reply:
I agree! (for me too that's enough)

> That's a good point. Yes, let's make the mention in the corresponding man page directly, that will be most useful to users.

Appreciate you for considering this route!
Since I based that upon what Toby wanted here:

> yes please I would like to see something on man pages in this case. (fread and fwrite)

And now that he agrees, I think we can move ahead on this consensus then :)